cli/src/multisig.rs

```diff
@@ -534,6 +574,24 @@ impl fmt::Display for ShowTransactionOutput {
                     "  Tried to deserialize a Solido instruction, but failed."
                 )?;
             }
+            ParsedInstruction::TokenInstruction(token_instruction) => {
+                write!(f, "  This is a Token instruction. ")?;
+                match token_instruction {
+                    TokenInstruction::Transfer {
+                        from_address,
+                        to_address,
+                        amount,
+                    } => {
+                        writeln!(f, "It transfers tokens owned by the multisig.")?;
+                        writeln!(f, "    From address: {}", from_address)?;
+                        writeln!(f, "    To address:   {}", to_address)?;
+                        writeln!(f, "    Amount:       {}", amount)?;
```

Comment:
Would be good to clarify that this is the amount of the smallest denomination. E.g. if you do it for stSOL, it’s not the amount of stSOL but the amount of stLamports.

The mint also stores the number of decimals, maybe we can use that to format, but on the other hand, maybe that makes it ambiguous too :confused: 

Just clarifying that it is the smallest denomination should work. Also, should we print the token mint address of these accounts, for convenience?

Reply:
I like printing the mint address, I'll add these changes.
Something like "Amount (in smallest denomination): " would work?

Reply:
:+1: yeah that works. If you like to keep then aligned, we could also do “Amount: X of the token’s smallest denomination”